system/ui/widgets/cameraview.py

```diff
@@ -0,0 +1,82 @@
+import pyray as rl
+from msgq.visionipc import VisionIpcClient, VisionStreamType
+from openpilot.system.ui.lib.application import gui_app
+
+FRAME_FRAGMENT_SHADER = """
+#version 330 core
+in vec2 fragTexCoord; uniform sampler2D texture0, texture1; out vec4 fragColor;
+void main() {
+  float y = texture(texture0, fragTexCoord).r;
+  vec2 uv = texture(texture1, fragTexCoord).ra - 0.5;
+  fragColor = vec4(y + 1.402*uv.y, y - 0.344*uv.x - 0.714*uv.y, y + 1.772*uv.x, 1.0);
+}"""
+
+
+class CameraView:
+  def __init__(self, name: str, stream_type: VisionStreamType):
+    self.client = VisionIpcClient(name, stream_type, False)
+    self.shader = rl.load_shader_from_memory(rl.ffi.NULL, FRAME_FRAGMENT_SHADER)
+    self.texture_y: rl.Texture | None = None
+    self.texture_uv: rl.Texture | None = None
+    self.frame = None
+
+  def __del__(self):
```

Comment:
I see `Segmentation fault (core dumped)` when I CTRL-C - can you repro? I think it's related to GL cleanup